pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -90,6 +90,12 @@ type MigPlanSpec struct {
 
 	// If set True, disables direct volume migrations.
 	IndirectVolumeMigration bool `json:"indirectVolumeMigration,omitempty"`
+
+	// IncludedResources optional list of included resources in Velero Backup
+	IncludedResources []*metav1.GroupKind `json:"includedResources,omitempty"`
```

Comment:
These must be optional as well, can we document what the default values are for this?